app/assets/stylesheets/_screen.scss

```diff
@@ -657,3 +650,43 @@ div#keycut-help {
 .btn-clipboard {
   padding: 4px 12px 5px;
 }
+
+.activity {
+  @extend .story;
```

Comment:
Can we avoid `@extend` here? It's bad to track and generates a CSS that is not gzip-friendly, can we create a mixin instead?